src/component/PokemonCard.js

```diff
@@ -0,0 +1,44 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import styles from "../component/PokemonCard.css"
+
+const PokemonCard = ({url}) => {
+    const [pokemon, setPokemon] = useState([])
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [pokemon, setPokemon] = useState();
```